### PR TITLE
Create ecs-ssh which forwards a command to docker containers

### PR DESCRIPTION
**Pull request by SamuelDavis**
https://www.pivotaltracker.com/story/show/121225757
The only thing I actually did was create the ecs-ssh alias and the awsGateway. But I formatted everything and broke all the multi-line aliases into separate files
aka
if pivotal-details or install or setup is bad, I don't care much and neither should you

### CODE REVIEW

File: aliases/install.sh [hunk 1]
```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env bash
```
Comment (sudo-reboot) [inline aliases/install.sh]: why did you move this file?

File: services/awsGateway.php [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+    private function awsCmd($cmd, array $options = [])
+    {
+        $options = array_merge($options, ['output' => 'json']);
+        foreach ($options as $key => $val) {
+            $cmd .= is_array($val) ? " --{$key} " . implode(' ', $val) : " --{$key}={$val}";
+        }
+
+        echo "aws {$cmd}\n";
+        return json_decode(shell_exec("aws {$cmd}"), true);
```
Comment (sudo-reboot) [inline services/awsGateway.php]: it would be nice if this worked with long running commands or even foreground commands like tail, i found this which might be useful http://stackoverflow.com/questions/20107147/php-reading-shell-exec-live-output

File: README.md [hunk 3]
```diff
@@ -3,3 +3,5 @@ run this
 ```bash
 curl -L https://raw.githubusercontent.com/daftlabs/dl-dev-profile/master/install.sh | bash
 ```
+
+To use any of the AWS commands (inc. ecs-ssh) you must install the aws cli and have an ~/.ssh/ecs.pem file which you can log into the various ec2 instance with.
```
Comment (sudo-reboot) [inline README.md]: can't we just do the ```$ curl "https://s3.amazonaws.com/aws-cli/awscli-bundle.zip" -o "awscli-bundle.zip"
$ unzip awscli-bundle.zip
$ sudo ./awscli-bundle/install -i /usr/local/aws -b /usr/local/bin/aws``` in the self update?

Comment (sudo-reboot) [inline README.md]: got that from http://docs.aws.amazon.com/cli/latest/userguide/installing.html

File: aliases/install.sh (hunk 1 above)
Comment (SamuelDavis) [inline aliases/install.sh]: It's an alias, I but it in the aliases directory. You may notice I've grouped all the commands in that big switch statement together into a directory called alliases

Comment (sudo-reboot) [inline aliases/install.sh]: its used by the update command, but its also the file that is referenced by the first installation command in the readme
